### DISCUSSION TITLE
A low resolution version of NorESM2.5

### DISCUSSION THREAD

**Discussion by adagj** [NorESMhub/noresm3_dev_simulations#35] [Discussions about coupled simulations]
Please list what features are wanted for a low resolution version of NorESM2.5 . 
Please share good arguments for having a low resolution versions and good arguments for only maintaining one version.
Please be pragmatic, but keep a friendly tone :)

**Comment by rosiealice** [comment]
(Good?) arguments for low -res are a lot to do with allowing parameter exploration (which is important),  investigation of alternative scenarios, longer scenarios (incl paleo) and initial condition ensembles. 

Arguments for only-one-version: Two versions are harder, some people are skeptical about low-res models, particularly wrt extreme events. 

Do both need to be ready on the same timescale?  Seems likely that the low res version would not be the CMIP7 version...

**Comment by MichaelSchulzMETNO** [comment]
From the discussion in Bergen:
- many required a lower resolution version of NorESM2.5
- Arguments for having a lower resolution mentioned were quicker science and development testing, paleo runs, possible bottlenecks at sigma2 if a higher resolved version would be very costly, potential to downscale lower resolution run to higher resolution with ML, less storage required
- Problems identified: Much more work to set up and tune a second version, likely no NCAR support for a 2 degree CAM and CTSM version,  
- Wall clock time gain should be estimated (eg 1 degree CAM&CTSM+ 1/4 BLOM versus 2 degree CAM&CTSM+ 1/2 BLOM)
- Unclear additional work amount needed for forcings & setup for low resolution version
-
